tools/src/main/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommandOptions.java

```diff
@@ -210,7 +216,9 @@ private ConsumerGroupCommandOptions(String[] args) {
     void checkArgs() {
         CommandLineUtils.maybePrintHelpOrVersion(this, "This tool helps to list all consumer groups, describe a consumer group, delete consumer group info, or reset consumer group offsets.");
 
-        CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt);
+        if (!options.has(validateRegexOpt)) {
+            CommandLineUtils.checkRequiredArgs(parser, options, bootstrapServerOpt);
+        }
```

Comment:
agree, but should we then update the bootstrap doc that states it's required?

Reply:
yeah, i considered doing this but i did not like it. how would you state it? "required except for --validate-regex"?

Reply:
it currently is "REQUIRED: The server(s) to connect to.". Maybe flipping it to "The server(s) to connect to. REQUIRED for all options except for --validate-regex"?

Reply:
works for me

Reply:
fixed